src/containers/support/SupportContactUsForm.tsx

```diff
@@ -0,0 +1,188 @@
+                  errorMessage={this.state.firstName.validation}
+                  label="First name"
+                  field={this.state.firstName}
+                  onValueChange={(field: IErrorableInput) => this.setState({ firstName: validatePresence(field, 'First Name') })}
+                  required={true} />
+              </div>
+              <div className="usa-width-one-half">
+              <ErrorableTextInput
+                errorMessage={this.state.lastName.validation}
+                label="Last name"
+                name="lastName"
+                field={this.state.lastName}
+                onValueChange={(field: IErrorableInput) => this.setState({ lastName: validatePresence(field, 'Last Name') })}
+                required={true} />
+            </div>
+          </div>
+          <div className="usa-grid">
+            <div className="usa-width-one-half">
+                <ErrorableTextInput
+                errorMessage={this.state.email.validation}
+                label="Email"
+                name="email"
+                field={this.state.email}
+                onValueChange={(field: IErrorableInput) => this.setState({ email: validateEmail(field) })}
+                required={true} />
+            </div>
+            <div className="usa-width-one-half">
+              <ErrorableTextInput
+                errorMessage={null}
+                label="Organization"
+                name="organization"
+                field={this.state.organization}
+                onValueChange={(field: IErrorableInput) => this.setState({ organization: field })}
+                required={false} />
+            </div>
+          </div>
+
+          <ErrorableCheckboxGroup
+            additionalFieldsetClass='va-api-checkboxes'
+            additionalLegendClass='va-api-contact-us-legend-description'
+            label='If applicable, please select any of the APIs pertaining to your issue.'
+            onValueChange={this.toggleApis}
+            id='default'
+            required={false}
+            options={this.apiOptions}
+            values={{ key: 'value' }}
+          />
+
+          <ErrorableTextArea
+            errorMessage={this.state.description.validation}
+            label="Please describe your question or issue in as much detail as you can provide. Steps to reproduce or any specific error messages are helpful if applicable."
+            onValueChange={(field: IErrorableInput) => this.setState({ description: validatePresence(field, 'Description') })}
+            name="description"
+            field={this.state.description}
+            required={true} />
+          
+        </fieldset>
+      </Form>
+    );
+  }
+
+  private get apiOptions(): object[] {
+    return Object.keys(apiDefs).map(api => {
+      return {
+        label: apiDefs[api].name,
+        value: api,
+      };
+    });
+  }
+
+  private get initialApiState() {
+    return Object.keys(apiDefs).reduce((accumulator, currentValue) => {
```

Comment:
```suggestion
    return Object.keys(apiDefs).reduce((accumulator, apiId) => {
```

something a little more specific if possible?